src/state/state.ts

```diff
@@ -2,19 +2,37 @@ import type { BigNumber } from 'ethers';
 import { produce, type Draft } from 'immer';
 
 import type { Config } from '../config/schema';
-import type { LocalSignedData, AirnodeAddress, TemplateId } from '../types';
+import type { chainId, dapiName, DecodedDataFeed, DataFeedId, SignedData } from '../types';
 
 interface GasState {
   gasPrices: { price: BigNumber; timestampMs: number }[];
   sponsorLastUpdateTimestampMs: Record<string, number>;
 }
 
+export interface DataFeedOnChainValue {
+  value: BigNumber;
+  timestamp: number; // in seconds
+}
+
+export interface DApi {
+  dataFeed: DecodedDataFeed;
+  dataFeedValues: Record<chainId, DataFeedOnChainValue>;
+  updateParameters: Record<chainId, UpdateParameters>;
+}
+
+export interface UpdateParameters {
+  deviationThresholdInPercentage: BigNumber;
+  deviationReference: BigNumber;
+  heartbeatInterval: number;
+}
+
 export interface State {
   config: Config;
   dataFetcherInterval?: NodeJS.Timeout;
   gasPriceStore: Record<string, Record<string, GasState>>;
-  signedApiStore: Record<AirnodeAddress, Record<TemplateId, LocalSignedData>>;
-  signedApiUrlStore: Record<string, Record<AirnodeAddress, string>>;
+  signedApiStore: Record<DataFeedId, SignedData>;
+  signedApiUrlStore: { url: string; lastReceived: number }[];
```

Comment:
Minor: could rename to `lastReceivedMs` to be in line with the other millisecond fields we have and separate from the on chain timestamps in seconds.